template.yaml

```diff
@@ -411,13 +378,13 @@ Resources:
         Type: String
 
 Outputs:
-  HttpApiHelloWorld:
-    Description: "API Gateway endpoint URL for deployed stage for Hello World function"
-    Value: !Sub "https://${HttpApi}.execute-api.${AWS::Region}.amazonaws.com/${Stage}/hello"
-  CustomDomainHttpApiHelloWorld:
-    Condition: shouldUseCustomDomainNames
-    Description: "API Gateway endpoint URL for deployed stage for Hello World function"
-    Value: !Sub "https://api${SubdomainSuffixParameter}.${DomainParameter}/hello"
+  # HttpApiHelloWorld:
```

Comment:
Should still output API path.  
Please replace `[...]ApiHelloWorld` with `[...]ApiCrowdactions` and `[...]/hello` with `[...]/crowdactions`.

Reply:
Done @rubenhorn